### PR TITLE
Command line interface for downloading files

### PR DESCRIPTION
**Pull request by bedroesb**
Hello,

A long time ago we discussed through e-mail about bringing CLI capabilities to the python package of OmicsDI.

I finally found the time to do so, and with this PR it is possible using only an accession number to download a dataset. 
- Use of the existing classes to do these things.
- Requirements a requirements file to be used in the setup.py. 
- Because I also give the version in the CLI, I centralized it in a _version.py file so it does not get out of sync .

I did not yet add tests or documentation.

Looking forward to your feedback!

### CODE REVIEW

**Comment by ypriverol** [comment]
@bedroesb can you check the PR errors.

**Comment by bedroesb** [comment]
@ypriverol I don't think it fails are related to my changes, `test_batch` for example give errors with the example that is supposed to work and `client.batch("gulugulu11", "momomomomo")` gives a different error as expected. I see that your latest commit also failed, aybe there is a higher problem?

**Comment by ypriverol** [comment]
@mindcrusher11 can you check this error?

**Comment by bedroesb** [comment]
@ypriverol The entrypoint is now `omicsdi` I don't know if you want to change this?

File: ddipy/dataset_client.py [hunk 1]
```diff
@@ -256,7 +256,7 @@ def batch(acc, database):
             raise BadRequest("missing parameter database", MISSING_PARAMETER, payload=None)
 
         res = requests.get(constants.BATCH_URL, params={
-            "acc": acc,
```
Comment (bedroesb) [inline ddipy/dataset_client.py:259]: @ypriverol changing "acc" to "accession" fixed the tests.

**Comment by bedroesb** [comment]
This is ready for review :)

**Review by ypriverol — APPROVED** [review]

**Pull request merged**